corehq/apps/repeaters/tests/test_repeater.py

```diff
@@ -302,6 +296,7 @@ def test_excluded_case_types_are_not_forwarded(self):
         CaseFactory(self.domain_name).post_case_blocks([non_white_listed_case])
         self.assertEqual(1, len(self.repeat_records(self.domain_name).all()))
 
+    # @run_with_all_backends
```

Comment:
we no longer store user_id with the transaction which repeaters use to determine who submitted the last case block. my guess is that we could get the user_id from the form that submitted it, but i'm not sure if that's always the case. it's also annoying because post_case_blocks doesn't have a user_id associated with the form